examples/database-seeding/arc.env.dev.yml

```diff
@@ -0,0 +1,8 @@
+services:
+  examples/database-seeding:latest:
+    debug:
+      path: .
+    ingress:
+      subdomain: api
+    parameters:
+      AUTO_DDL: migrate
```

Comment:
@davidthor @tjhiggins 

Do we have open thoughts on environment config naming conventions when we have several of them next to each other like this?

For this demo I called them this:
```
arc.env.dev.yml
arc.env.qa.yml
arc.env.prod.yml
```

Would like to settle on something even if it's not this so we're consistent.

Reply:
It's intuitive enough. Fine for now and other examples.